src/labels_by_user_input.py

```diff
@@ -19,19 +23,39 @@ def labels_by_user_input(event_data, pull):
         pull=pull
     ):
         add_label(pull=pull, label=LABEL_VERIFIED)
-        set_commit_status_success_verify(commit=last_commit)
+        set_commit_status(
```

Comment:
Love the generic method, but any reason we shouldn't reduce this to:
```
last_commit.create_status(...)
```
?

Reply:
@sarahbx what do you suggest? To have last_commit class? or I missunderstood you?

Reply:
Sorry I was not clear.
`set_commit_status(commit=,...)` has no functionality other than as a wrapper for `commit.create_status(...)`
Do we require the wrapper?

Reply:
Well, you are right :) 
wrapper that do nothing is always unwanted.

Reply:
done